configs/opencv-ovms/instance_segmentation_demo/python/common/python/openvino/model_zoo/model_api/models/faceboxes.py

```diff
@@ -0,0 +1,142 @@
+"""
```

Comment:
Same comment as before. Will the demo still work if we remove some of the unreferenced common python files.

Reply:
I can refactor it in a different PR